### PR TITLE
Ws week picker

### PR DESCRIPTION
**Pull request by faboweb**
Pull request for issue: #30

Component Screenshot:
image

Description (short): 
Component was broken because of some typos. + Added new styling by UX.

### CODE REVIEW

**Comment by faboweb** [comment]
I don't know why, but there is some foreign commit in there ^^

File: src/ws-week-picker/ws-week-picker.scss [hunk 1]
```diff
@@ -15,8 +16,10 @@ $padding: 1rem;
     }
   }
   @if $bottom != 0 {
-    tr:first-child:after {
-      height: $bottom;
+    tr:last-child {
+      td, th {
+        padding-bottom: $bottom !important;
```
Comment (fragsalat) [inline src/ws-week-picker/ws-week-picker.scss]: Just want to not that using !important is not the best solution :)

Comment (faboweb) [inline src/ws-week-picker/ws-week-picker.scss]: valid point. maybe I change it tomorrow.

**Comment by myueksel** [comment]
looks great!

**Pull request merged**